client/state/timezones/reducer.js

```diff
@@ -27,8 +32,15 @@ export const byContinents = createReducer( {}, {
 	[ TIMEZONES_RECEIVE ]: ( state, actions ) => ( actions.byContinents )
 }, continentsSchema );
 
+export const requesting = createReducer( false, {
+	[ TIMEZONES_REQUEST ]: () => true,
+	[ TIMEZONES_REQUEST_FAILURE ]: () => false,
+	[ TIMEZONES_REQUEST_SUCCESS ]: () => false
+} );
+
 export default combineReducers( {
 	rawOffsets,
 	labels,
 	byContinents,
+	requesting
```

Comment:
can we use more descriptive naming here? `isRequesting` indicates that it's some boolean flag indicating the process while `requesting` could connote different information as well, such as details about a request that's happening.